pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -58,11 +64,28 @@ Examples:
 				newBuild = buildutil.GenerateBuildFromBuild(build)
 			}
 
+			var wg sync.WaitGroup
+			if streamLogs {
+				rd, err := client.Get().Namespace(namespace).Prefix("redirect").Resource("buildLogs").Name(buildName).Stream()
```

Comment:
there shouldn't be, which is why i'm so confused..your pastebin implied this was working, but i'm not sure how :)

the terminology is a bit wonky, but you use a buildconfig to create a build, so if you start the same buildconfig twice, you'll get two unique build objects created (note the distinction of buildconfig vs build), each with their own pod, own logs, etc.